File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.py
class Solution(object):
    def lastElement(self,n):
        sum=0
        while n>0:
            sum+=n%10
            n=n//10
        return sum
    def minElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        arr=list(map(self.lastElement,nums))
        min=arr[0]
        for i in arr:
            if i<min:
                min=i
        return min

File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.cpp
class Solution {
public:
    int minElement(vector<int>& nums) {
        int min=nums[0];
        for(int i : nums){
            int sum=0;
            while(i>0){
                sum+=i%10;
                i=i/10;
            }
            if(sum<min){
                min=sum;
            }
        }
        return min;
    }
};